server/graphql_schemas/event/schema.py

```diff
@@ -123,6 +126,94 @@ def mutate_and_get_payload(cls, root, info, **input):
         return cls(action_message="Event deactivated")
 
 
+class SendEventInvite(relay.ClientIDMutation):
+    message = graphene.String()
+
+    class Input:
+        event_id = graphene.Int(required=True)
+        receiver_email = graphene.String(required=True)
+
+    @classmethod
+    def mutate_and_get_payload(cls, root, info, **input):
+        event_id = input.get('event_id')
+        sender = AndelaUserProfile.objects.get(
+            user_id=info.context.user.id or 3)
```

Comment:
Well done @tomipaul.  But please what's `3` in line 140 for?

Reply:
@tosmak16 that's just for some manual testing; forgot to remove it after.